Tooll/Components/TextButtonEdit.xaml.cs

```diff
@@ -125,6 +132,14 @@ protected void TextEdit_LostFocus(object sender, RoutedEventArgs e)
             }
             TextEdit.Visibility = Visibility.Collapsed;
         }
+        
+        protected void TextEdit_PreviewMouseDown(object sender, MouseButtonEventArgs e)
+        {
+            if (e.ClickCount == 3)
+            {
+                XTextEdit.SelectAll();

```

Comment:
@pixtur what would be the point of adding a return there? no value to return and method returns right there anyway...